### PR TITLE
Update the success email

### PR DESCRIPTION
**Pull request by tracymu**
image

image

### CODE REVIEW

**Comment by lbain** [comment]
Pro tip: https://chrome.google.com/webstore/detail/full-page-screen-capture/fdpohaocaechififmbbbbbknoalclacl lets you take multi-screen screenshots :)

File: app/views/applications_mailer/application_success.html.erb [hunk 1]
```diff
@@ -312,31 +312,34 @@ Rails Girls has proven very popular, and we have more applicants than spots. Ple
...
+
+<p>The workshop will start promptly at 9am, so please make sure you arrive well before then. You'll be using your laptop all day, so don't forget to bring it along with you. The day will end around 5pm.</p>
+
 &nbsp;
 <h4 class="null" style="color: #202020;display: block;font-family: Arial;font-size: 22px;font-weight: bold;line-height: 100%;margin-top: 0;margin-right: 0;margin-bottom: 10px;margin-left: 0;text-align: left;">Location</h4>
-Rails Girls is being held at Freelancer, which is at Level 20, 680 George Street, Sydney, New South Wales, 2000</a>. If you arrive late and the door isn&#39;t open, please call me (Angie) on 044-706-8945) and someone will pop down to let you in. Please be on time so we don&#39;t spend too much time on the lift!<br>
+Rails Girls is being held at Canva, which is at 1/2 Lacey Street, Surry Hills</a>. If you arrive late and the door isn&#39;t open, please call me (Tracy) on 0434-940-141) and someone will pop our to let you it! (But please try hard to be on time)<br>
```
Comment (angelamikulasev) [inline app/views/applications_mailer/application_success.html.erb]: typo - `someone will pop out to let you in!`

File: app/views/applications_mailer/application_success.html.erb [hunk 2]
```diff
@@ -312,31 +312,34 @@ Rails Girls has proven very popular, and we have more applicants than spots. Ple
...
+<a href="http://www.lookahead.com.au/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">LookAhead Search</a><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.envato.com/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Envato</a></span><br>
+<a href="https://cultureamp.com/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Culture Amp</a></span><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.rea-group.com" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">REA</a></span><br>
+<span style="font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px"><a href="https://www.hired.com" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">Hired</a></span><br>
 <br>
 We will be limiting the amount of email that you will get from us to the bare essentials, please make sure to watch your email for any updates.<br>
```
Comment (angelamikulasev) [inline app/views/applications_mailer/application_success.html.erb]: These are just suggestions, but perhaps something along the lines of 
> We will be limiting the amount of emails being sent to you so please be sure to watch your inbox for any updates.

or 

> We will be limiting the amount of emails being sent to you so please be sure to keep an eye out

File: app/views/applications_mailer/application_success.html.erb [hunk 3]
```diff
@@ -312,31 +312,34 @@ Rails Girls has proven very popular, and we have more applicants than spots. Ple
...
 &nbsp;<br>
-We have a twitter account at <a href="https://twitter.com/railsgirls_syd" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">https://twitter.com/railsgirls_syd</a>&nbsp;and a Facebook group at <a href="http://www.facebook.com/groups/474567595959477/" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">http://www.facebook.com/groups/474567595959477/</a><br>
-Please follow/join us to get updates in the future.<br>
+We have a twitter account at <a href="https://twitter.com/railsgirls_syd" target="_blank" style="color: #336699;font-weight: normal;text-decoration: underline;">@railsgirls_syd</a> that you might like to follow as well.<br>
 &nbsp;<br>
-If you have any questions, please don&rsquo;t hesitate to contact us at railsgirlssyd@gmail.com. Otherwise we look forward to seeing you on Friday the 9th.<br>
+If you have any questions, please don&rsquo;t hesitate to contact us at railsgirlssyd@gmail.com. Otherwise we look forward to seeing you on Saturday 27th May.<br>
 &nbsp;<br>
 Best,<br>
```
Comment (angelamikulasev) [inline app/views/applications_mailer/application_success.html.erb]: should this be "all the best" ?

File: app/views/applications_mailer/application_success.html.erb [hunk 4]
```diff
@@ -286,7 +286,7 @@
...
 
-<p style="margin-right: 0px; margin-bottom: 15px; margin-left: 0px; color: rgb(0, 0, 0); font-family: Helvetica, arial, freesans, clean, sans-serif; font-size: 15px; line-height: 25.5px; margin-top: 0px !important;">Congratulations! You&#39;re invited to join the fifth Rails Girls (and third Rails Girls Next) workshop in Sydney. <span style="color:rgb(0, 0, 0); font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px">We hope this workshop will offer you a great introduction to the wonderful world of building web applications.</span><br>
+<p style="margin-right: 0px; margin-bottom: 15px; margin-left: 0px; color: rgb(0, 0, 0); font-family: Helvetica, arial, freesans, clean, sans-serif; font-size: 15px; line-height: 25.5px; margin-top: 0px !important;">Congratulations! You&#39;re invited to join the sixth Rails Girls (and fourth Rails Girls Next) workshop in Sydney. <span style="color:rgb(0, 0, 0); font-family:helvetica,arial,freesans,clean,sans-serif; font-size:15px; line-height:25.5px">We hope this workshop will offer you a great introduction to the wonderful world of building web applications.</span><br>
```
Comment (angiegove) [inline app/views/applications_mailer/application_success.html.erb]: I think this is the 9th event in Sydney

**Pull request merged**